docs/dev/install.rst

```diff
@@ -194,3 +194,74 @@ If you ever need to bypass Therapist, you can do so by passing
 ``--no-verify`` to your ``git commit`` command.
 
 .. _Therapist: http://therapist.readthedocs.io/en/latest/overview.html
+
+.. _remotesettings-install:
+
+Remote Settings
+---------------
+If you want to enable the publication of recipes on `Remote Settings`_, you'll need to set up
+an instance locally.
+
+1. Follow the `Remote Settings installation instructions`_ to launch a development
+   instance of Remote Settings locally.
+
+2. Configure Normandy to enable the integration by adding the following lines
+   to ``.env`` (create the file if it does not exist yet):
+
+   .. code-block:: ini
+
+      DJANGO_REMOTE_SETTINGS_URL=http://localhost:8888/v1
+      DJANGO_REMOTE_SETTINGS_USERNAME=normandy
+      DJANGO_REMOTE_SETTINGS_PASSWORD=n0rm4ndy
+
+3. Adjust the Remote Settings configuration in ``server.ini`` for the Normandy
+   recipes collection, and restart the Remote Settings container:
+
+   .. code-block:: ini
+
+      kinto.signer.main-workspace.normandy-recipes.to_review_enabled = false
+      kinto.signer.main-workspace.normandy-recipes.group_check_enabled = false
+
+4. Create the dedicated Remote Settings user and collection for Normandy:
+
+   .. code-block:: bash
+
+      SERVER=http://localhost:8888/v1
+
+      curl -X PUT ${SERVER}/accounts/normandy \
+           -d '{"data": {"password": "n0rm4ndy"}}' \
+           -H 'Content-Type:application/json'
+
+      curl -X PUT ${SERVER}/buckets/main-workspace \
+           -H 'Content-Type:application/json' \
+           -u 'normandy:n0rm4ndy'
+
+      curl -X PUT ${SERVER}/buckets/main-workspace/collections/normandy-recipes \
+           -H 'Content-Type:application/json' \
+           -u 'normandy:n0rm4ndy'
+
+With both configurations in place, Normandy should start without error. When a recipe is enabled, it is published and becomes visible
+on the Remote Settings server:
+
+.. code-block:: bash
+
+      curl ${SERVER}/buckets/main/collections/normandy-recipes/records
+
+On the Firefox side, pulling the recipes from the local instance of Remote Settings will require these preferences to
+be set:
+
+- ``services.settings.server``: ``http://localhost:8888/v1``
+- ``services.settings.verify_signature``: false
```

Comment:
I don't have this. 
<img width="1101" alt="screen shot 2018-11-30 at 3 59 58 pm" src="https://user-images.githubusercontent.com/26739/49314637-0bdd9c00-f4b9-11e8-9179-f37123fd7b6a.png">